tests/functional/commands/conftest.py

```diff
@@ -441,3 +447,111 @@ def three_experiments_same_name_with_trials(
         orionstate.database.write("trials", trial2.to_dict())
         orionstate.database.write("trials", trial3.to_dict())
         x_value += 1
+
+
+@pytest.fixture
+def three_experiments_family_same_name_with_trials(
```

Comment:
To be coherent with other tests, that's the `three_experiments_branch_same_name`.

Reply:
Maybe `benchmark` should be in the name of the test as well since this fixture is the only one adding adding a benchmark to the DB.

Reply:
Ok!